OnlineShopAPI/OnlineShopAPI.sln

```diff
@@ -1,11 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.1.32228.430
+# Visual Studio Version 16
```

Comment:
Not related to PR, but it's better to update VS for us to have the same dev environment

Reply:
hm why version 16 because i have 19

Reply:
no, actually 16, i started updating